### ISSUE TITLE
Identify not working?

### ISSUE THREAD

**Issue by emergingdzns** [GeneaLabs/laravel-mixpanel#6]
Sorry to keep pounding on your library, but hopefully this all helps.

I've got the library very nicely tracking the page loads and custom events, but for some reason the user info isn't getting submitted to MixPanel.

I'm submitting it like this:
```
$user = Auth::user();
$this->mixPanel->identify($user->user_id,[
      'customer_id' => $user->customer_id,
      'name' => $user->name
]);
```
I verified that the $user object is in fact valid and that user_id, customer_id, and name all can echo out onto the page. Not sure what to do next.

Thanks again!

**Comment by mikebronner** [comment]
Hi @emergingdzns, no this is great! :) keep bringing up the choke points you are finding. I will take a look. 

User info is only submitted when the user logs in. You should not have to manually track the user, it should already be done for you. Let me get back to you on this in a bit.

**Comment by emergingdzns** [comment]
Ok. I understand the user data getting logged on login but in the mixpanel ui I'm seeing all anonymous users so I added that bit to the actual custom event tracking. We should at least be submitting user id which I think might actually be happening, but it's hard to tell in Mixpanel. I'll log out and back in to see if that fixes things.

**Comment by emergingdzns** [comment]
Ok so logging out is an issue. Our system doesn't have first name and last name, only name. So I got the following:
```
QueryException in Connection.php line 636: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'first_name' in 'field list' (SQL: update `users` set `remember_token` = a8kyWZxsQLDJLXe9naBTsoT4rw3qd77q4s9LLupjQBa8slK7ISQJN2clHo10, `first_name` = Andrew, `last_name` = Christensen where `user_id` = 54)
```
What's odd is this doesn't mention the mixpanel file anywhere but we don't have any code for updating on logout that I'm aware of.

When I logged back in I had no issues, but in Mixpanel there's still no user data. It also isn't showing the login either so I'm guessing our lack of first/last is messing things up.

That all being said, we can transition to a first/last structure as we don't have a ton of users already.

**Comment by mikebronner** [comment]
Ah, that's true ... I can update for $name vs. $first_name and $last_name (will just split the name by spaces, and the last word will be the last name, the rest the first name).

**Comment by emergingdzns** [comment]
That works. Thanks again. This is a pretty cool library and saved us a bunch of time.

**Comment by mikebronner** [comment]
I think the issue here was that I was adding the first_name and last_name columns to your user instance, and that broke things. I updated the mix panel events not to manipulate your user class, and this should now fix things in regards to the error. This will be fixed in version 0.4.6, coming out in the next few minutes.

We should open a separate issue regarding mix panel not tracking users.

**Comment by mikebronner** [comment]
@emergingdzns If tracking on MixPanel UI still isn't showing any user details, I will be happy to jump on ScreenHero with you and see what the issue might be. Let me know after you update how it works. (You should be able to update now to 0.4.6 running `composer update`.)

**Comment by emergingdzns** [comment]
@mikebronner Sorry for the delay. I had to put this on hold temporarily to deal with some other stuff. I'm back on it now and I'm still not getting any user data in MixPanel. I get "anonymous" users.

Each time a report in our system is run I execute the lines I posted when I created this issue. I then run this:
```
$this->mixPanel->track('Our Report Name', Input::all());
```
Right now, I am just dumping the input params to the tracker just to see waht we get. Eventually we'll change this up to upload only the data we want in the track function.

Would certainly be willing to do a screenshare with you so we can try to figure this out.

Thanks!
